common/web/lm-worker/build-polyfiller.js

```diff
@@ -4,6 +4,71 @@ import SourceMapCombiner from 'combine-source-map';
 import convertSourceMap from 'convert-source-map'; // Transforms sourcemaps among various common formats.
                                                    // Base64, stringified-JSON, end-of-file comment...
 
+import esbuild from 'esbuild';
+
+let sourceFromArgs;
+let destFromArgs;
+
+function doHelp(errCode) {
```

Comment:
Suggest making this:

```suggestion
function doHelp(errMessage) {
  if(errMessage) {
    console.error(errMessage + '\n');
  }
```